File: fix_settings.js
const fs = require('fs');
let content = fs.readFileSync('apps/pos/src/pages/AdminSettings.tsx', 'utf8');
content = content.replace(
    "allowSelfRegistration: true, pairingSecret: '', linkedDevices: []",
    "allowSelfRegistration: true, pairingSecret: '', deviceNotes: '', linkedDevices: []"
);
fs.writeFileSync('apps/pos/src/pages/AdminSettings.tsx', content);
console.log('Fixed AdminSettings.tsx');

File: fix_settings.py
import re
with open('apps/pos/src/pages/AdminSettings.tsx', 'r', encoding='utf-8') as f:
    content = f.read()
content = content.replace(
    "allowSelfRegistration: true, pairingSecret: '', linkedDevices: []",
    "allowSelfRegistration: true, pairingSecret: '', deviceNotes: '', linkedDevices: []"
)
with open('apps/pos/src/pages/AdminSettings.tsx', 'w', encoding='utf-8') as f:
    f.write(content)
print('Fixed AdminSettings.tsx')